homeassistant/components/deconz/device_trigger.py

```diff
@@ -91,6 +91,14 @@
     (CONF_LONG_RELEASE, CONF_TURN_OFF): {CONF_EVENT: 4003},
 }
 
+HUE_BUTTON_REMOTE_MODEL = "Hue smart button"
```

Comment:
Not sure that's the right identifier for deCONZ... have you search for it in the deconz libraries?

Reply:
Any pointers on where to look for it? I'm not too familiar with deCONZ.

Reply:
I'll take a look, thanks.

Reply:
Got it - I've updated this with the correct modelid and productname now!